### PR TITLE
Add GToTree v1.7.05

### PR DESCRIPTION
**Pull request by vinisalazar**
Hi,

this recipe adds [GToTree](https://github.com/AstrobioMike/GToTree/).

Thanks,
V

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by vinisalazar** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

File: recipes/gtotree/post-link.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+printf "will cite" | parallel --citation 2&> /dev/null
+
+# removing coreutils ls so the user-system one is used still
+rm ${CONDA_PREFIX}/bin/ls
```
Comment (dpryan79) [inline recipes/gtotree/post-link.sh:4]: Is this strictly needed? We really try to avoid modifying other packages.

**Review by dpryan79 — CHANGES_REQUESTED** [review]
One minor question in post-link.sh

**Comment by vinisalazar** [comment]
@AstrobioMike thoughts?

File: recipes/gtotree/post-link.sh (hunk 1 above)
Comment (AstrobioMike) [reply recipes/gtotree/post-link.sh:4]: Heya,

Yea, it ruins people's ls settings if I don't remove it. I need some things i can easily get from the coreutils package, i just don't want to add another ls program in front of the user's regular one.

**Comment by AstrobioMike** [comment]
Sorry, @vinisalazar, this shouldn't be a noarch build. I build and package for osx-64 and linux-64. It relies on a unix-like command line

**Comment by dpryan79** [comment]
The package itself is correctly noarch, since it itself doesn't have any architecture-dependent compilation. Dependencies can be architecture dependent without issue.

**Review by dpryan79 — APPROVED** [review]

**Comment by vinisalazar** [comment]
Thank you @dpryan79 !

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by AstrobioMike** [comment]
> The package itself is correctly noarch, since it itself doesn't have any architecture-dependent compilation. Dependencies can be architecture dependent without issue.

Thank you, @dpryan79 :) (and for all you do here for bioconda!)

I still have a question/request though. Even if the compilation isn't architecture-dependent, the package is operating-system dependent (meaning gtotree itself is operating-system dependent, not necessarily a dependency of it – this isn't a python package, it's basically a shell package with also lots of python, ha). Right now, since bioconda has a noarch version, it is findable and can be attempted to be installed on a windows conda. When maintaining specifically just linux and osx versions (like i've been doing [on my channel](https://anaconda.org/AstrobioMike/gtotree)), rather than a noarch, it's not findable on a windows conda, which is I think more clear/less misleading. Can you tell me how can we set it up so that there are linux and osx versions maintained, and not a noarch version?

**Pull request merged**